components/Profile/ProfileCard.tsx

```diff
@@ -0,0 +1,147 @@
+                            <span className={profileCard.userProfileName}>
+                                {name}
+                            </span>
+                            <div
+                                className={
+                                    'flex items-center justify-self-end space-x-2'
+                                }
+                            >
+                                {uid !== currentUserUid ? (
+                                    <>
+                                        <Button
+                                            onClick={() => {
+                                                alert('TODO: Follow')
+                                            }}
+                                            text={'Follow'}
+                                            keepText={true}
+                                            addStyle={profileCard.newPostButton}
+                                        />
+                                    </>
+                                ) : (
+                                    <>
+                                        <Button
+                                            onClick={openModal}
+                                            text={'Edit Profile'}
+                                            type={'button'}
+                                            addStyle={profileCard.editButton}
+                                            keepText={true}
+                                        />
+                                        {!isMobile && (
+                                            <Button
+                                                onClick={() => {
+                                                    alert('TODO: Add new post')
+                                                }}
+                                                icon={
+                                                    <UilPen
+                                                        className={
+                                                            'h-5 w-5 mr-1'
+                                                        }
+                                                    />
+                                                }
+                                                text={'New Post'}
+                                                addStyle={
+                                                    profileCard.newPostButton
+                                                }
+                                            />
+                                        )}
+                                    </>
+                                )}
+                            </div>
+                        </div>
+                        {/*username*/}
+                        {username && (
+                            <span className={profileCard.usernameText}>
+                                @{username}
```

Comment:
Here I'm not sure about the '@' which is a straight copy from Twitter.
I don't see the point of having it or is there a reason I don't see?